.github/workflows/_linux-build.yml

```diff
@@ -38,7 +38,7 @@ jobs:
   build:
     # Don't run on forked repos.
     if: github.repository_owner == 'pytorch'
-    runs-on: [self-hosted, linux.2xlarge]
+    runs-on: [self-hosted, linux.4xlarge]
```

Comment:
Why this is needed for NCCL update?

Reply:
Please see the comments above. Building on `2xlarge` ran into the `virtual memory exhausted: Cannot allocate memory` error as @mrshenli mentioned.